FrontEnd/src/components/RestorePassword/components/restorepassword/restorepassword-form/EmailFormContent.js

```diff
@@ -0,0 +1,102 @@
+import { useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import { toast, ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import axios from 'axios';
+import styles from './EmailFormContent.module.css';
+
+export function SendEmailRestorePasswordFormContentComponent({ setIsValid }) {
+  const navigate = useNavigate();
+
+  const errorMessageTemplates = {
+    required: 'Обов’язкове поле',
+    email: 'Email не відповідає вимогам',
+  };
+
+  const emailPattern = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+
+  const {
+    register,
+    handleSubmit,
+    getValues,
+    formState: { errors, isValid },
+  } = useForm({
+    mode: 'all',
+  });
+
+  useEffect(() => {
+    const formIsValid = isValid;
+    setIsValid(formIsValid);
+  }, [isValid, setIsValid]);
+
+  const onSubmit = () => {
+    const dataToSend = {
+      email: getValues('email'),
+    };
+
+    axios({
+      method: 'post',
+      url: `${process.env.REACT_APP_BASE_API_URL}/api/auth/users/reset_password/`,
```

Comment:
Do we still need to use this env var? I thought that we decided to use the relative path since API leaves on the same domain

Reply:
I do not have any information about it